index.html

```diff
@@ -18,10 +18,37 @@
 <body>
   <header>
     <div class="container">
+      <h1>Code Journal</h1>
     </div>
   </header>
   <main class="container">
-    <div data-view="entry-form"></div>
+    <div data-view="entry-form">
+      <form action="">
+        <h1>New Entry</h1>
+        <div class="col-md">
+          <div class="form-image-wrapper">
+            <img src="images/placeholder-image-square.jpg" alt="placeholder image" class="image">
+          </div>
+          <div class="input-container-md">
+            <div class="input-wrapper">
+              <label for="title">Title</label>
+              <input type="text" name="title" id="title" required>
+            </div>
+            <div class="input-wrapper">
+              <label for="photo-url">Photo URL</label>
+              <input type="text" name="photo-url" id="photo-url" required>
+            </div>
+          </div>
```

Comment:
I would use the layout classes we already provided for you. Just give this a `column-half`